src/apis/axios.ts

```diff
@@ -1,11 +1,50 @@
 import axios from 'axios';
+import Cookies from 'js-cookie';
+
+import { ACCESS_TOKEN_EXPIRED_TIME, REFRESH_TOKEN_EXPIRED_TIME } from '@/constants';
+
+import { Auth } from './auth';
 
 const instance = axios.create({
   baseURL: process.env.NEXT_PUBLIC_BASE_URL,
   timeout: 5000,
   headers: {
     'Content-Type': 'application/json',
   },
+  withCredentials: false,
 });
 
+instance.interceptors.request.use(
+  async (config) => {
+    const accessToken = Cookies.get('accessToken');
+    const refreshToken = Cookies.get('refreshToken');
+    if (!accessToken && refreshToken) {
+      try {
+        const res = await Auth.renewToken();
+        const { accessToken, refreshToken } = res.data;
```

Comment:
```suggestion
        const response = await Auth.renewToken();
        const { accessToken, refreshToken } = response.data;
```
response로 풀어서 사용하면 더 좋을 거 같습니다!

Reply:
```suggestion
        const { data } = await Auth.renewToken();
        const { accessToken, refreshToken } = data;
```

전 아예 res도 안쓰고 바로 data를 뽑아쓰는걸 더 선호합니당